syncstorage-postgres/migrations/2025-10-20-155711_create_schema/up.sql

```diff
@@ -0,0 +1,131 @@
+-- fxa_uid: a 16 byte identifier, randomly generated by the fxa server
+--    usually a UUID, so presuming a formatted form.
```

Comment:
It is a UUIDv4 value.  The FxA tables store that in binary.  The textual representation are usually lowercase, without the '-' chars.